### PR TITLE
README.md: Small tweaks to spelling/whitespace

### PR DESCRIPTION
**Pull request by msabramo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Docverter Server
 
-Docverter is a document conversion server with an HTTP interface. It wrap the following open source software in a JRuby app:
+Docverter is a document conversion server with an HTTP interface.
+It wraps the following open -ource software in a JRuby app:
```
Comment (peterkeen) [inline README.md]: What is `-ource`?

Comment (msabramo) [inline README.md]: Oops. :smile: Just updated.

File: Gemfile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 source 'http://rubygems.org'
 
-ruby '1.9.3', engine: 'jruby', engine_version: '1.7.4'
+ruby '1.9.3'  # , engine: 'jruby', engine_version: '1.7.4'
```
Comment (peterkeen) [inline Gemfile]: This is not correct. It only coincidentally works on MRI if you're not using the PDF conversions.

Comment (msabramo) [inline Gemfile]: Oops. I committed that accidentally. 

Was trying to run locally on my laptop since I don't have a Heroku account handy. I'll remove.

**Pull request merged**